pkg/exporter/metrics.go

```diff
@@ -67,6 +67,11 @@ func (e *Exporter) metricsHandler(scriptName string, params url.Values, promethe
 	// Get env vars
 	runEnv := e.Config.GetRunEnv(scriptName)
 
+	for key, value := range params {
+		last_value := value[len(value)-1]
+		runEnv[key] = last_value
+	}
```

Comment:
HI @jonasbadstuebner, thanks for your contribution 🙂. Can we maybe check if the `runEnv` map already contains a key and only add it if it is not present, to potentially break it for users which have params and environment variables with the same name?

```suggestion
	for key, value := range params {
		if _, ok := runEnv[key]; !ok {
			last_value := value[len(value)-1]
			runEnv[key] = last_value
		}
	}
```

Out of curiosity, why are you using the last param value and not the first one?

Reply:
Would it make sense to make this configurable? This would give the possibility to set default env vars but overwrite them for different scenarios, but only if explicitly stated.

To be honest, I can only guess right now, why I chose to use the last value. I don't remember anymore.
I think it was to mimic the env behavior where variables that are defined later overwrite the first ones.
Maybe joining them together with commas would make even more sense here? What do you think?
This way I could access all values in a script and not just the first one.

Reply:
I "hijacked" the ping script to show the usage of my change.
You can open `http://localhost:9469/probe?script=ping&target_ips=10.0.0.0,127.0.0.1` for example and it is going to error on the first IP, even though the second one would work.

It also works with specifying the parameter multiple times, instead of already supplying a comma separated list: `http://localhost:9469/probe?script=ping&target_ips=8.8.8.8&target_ips=127.0.0.1`

And setting `allowEnvOverwrite` to `false` only pings 127.0.0.1 each time.

Reply:
Awesome thank you very much 🙂